src/game/default/g_client_stats.c

```diff
@@ -54,6 +54,11 @@ void G_ClientToIntermission(g_entity_t *ent) {
 
 	ent->client->locals.ammo_index = 0;
 	ent->client->locals.pickup_msg_time = 0;
+	
+	// take a screenshot if we're supposed to
+	if (g_force_screenshot->integer == 1){
+		G_ClientStuff(ent, "r_screenshot\n");
+	}
```

Comment:
I think I'd rather see this implemented as a client-side latch with a cvar to enable / disable. The game could send a new command (`SV_CMD_INTERMISSION`), which the cgame could parse and, if the appropriate cvar is enabled (`cg_intermission_screenshot`?), then take the shot. Thoughts?